addons/mail/static/src/core/common/suggestion_service.js

```diff
@@ -166,11 +166,28 @@ export class SuggestionService {
                 suggestions.push(partner);
             }
         }
+        const specialSuggestions = this.store.specialMentions
+            .filter((special) => {
+                return (
+                    special.channel_types.includes(thread.channel_type) &&
+                    cleanedSearchTerm.length >= Math.min(4, special.label.length) &&
+                    (special.label.startsWith(cleanedSearchTerm) ||
+                        cleanTerm(special.description.toString()).includes(cleanedSearchTerm))
+                );
+            })
```

Comment:
Nitpicking: you could avoid an indent level by using `=> (/* expr */)` rather than `=> { return /* expr */ }`